sklearn/compose/tests/test_target.py

```diff
@@ -265,3 +267,29 @@ def test_transform_target_regressor_ensure_y_array():
     tt.predict(X.tolist())
     assert_raises(AssertionError, tt.fit, X, y.tolist())
     assert_raises(AssertionError, tt.predict, X)
+
+def test_transform_target_regressor_allow_nan():
+    
+    # check if the TransformedTargetRegressor allows missing value in the target array
+    X, y = datasets.load_linnerud(return_X_y=True)   
+    y[5, 1] = np.NaN
+    
+    check_array(y, accept_sparse=False, force_all_finite=False,
```

Comment:
I don't think this should be in a test

Reply:
It's still unresolved, pressed the resolved button by mistake.